src/main/java/frc/robot/controls/Driver.java

```diff
@@ -41,4 +46,48 @@ public Driver(Robot robot)
 	// METHODS //
 
 	// Add methods here which return values for various robot controls by reading the controllers.
+
+    /* =====================================
+    getThrottle() will return the current
+    value of the Y-Axis of the left stick on the controller
+    It will also handle adjusting
+    the reading according to the deadzone,
+    and according to the max throttle speed.
+    ===================================== */
+    public double getThrottle()
+    {
+        double throttleValue = 0;
+        
+        throttleValue = -dualshock.getY();
+        throttleValue *= Constants.MAX_THROTTLE_SPEED;
+
+        if (Math.abs(throttleValue) < Constants.DEADZONE)
```

Comment:
Be careful here, you've already updated `throttleValue` to be a fraction of `MAX_THROTTLE_SPEED` which I am assuming is the speed you are setting to the motors, but you are comparing it to the controller deadzone? If that `DEADZONE` is referring to the raw controller value from `dualshock.getY()`, you should be checking that in a separate if-statement before line 61 to immediately return if controller reading is in deadzone Ex `if(Math.abs(dualshock.getY()) < Constants.DEADZONE) return 0;`

Otherwise, right now the robot will not move until you hit a throttleValue of more than 0.15